docs/DeveloperGuide.adoc

```diff
@@ -326,41 +326,281 @@ _{More to be added}_
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `$AVE IT` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: Add
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to add a spending. Format: `Add -i INFO -a AMT [-t tag] [-d DATE] `
+2.  `$AVE IT` will acknowledge and assign this expenditure an unique ID.
+3.  If tag is not specified during the instruction, `$AVE IT` will auto assign it to be `others`
 +
+
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1. Wrong Format.
++
+Use case back to step 1.
+
+[discrete]
+=== Use case: Delete
+
+*MSS*
+
+1.  User requests to delete a spending. Format: `Delete UniqueID`
+2.  `$AVE IT` will acknowledge.
+3.  If tag is not specified during the instruction, `$AVE IT` will auto assign it to be `others`
++
+
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1. Wrong Format.
+* 2. Invalid ID.
++
+Use case back to step 1.
+
+[discrete]
+=== Use case: Repeat
+
+*MSS*
+
+1.  User requests to add an fix spending per month. Format: `Repeat -i INFO -a AMT -d DATE -p INTERVAL MONTH [-t TAG]`
```

Comment:
Format : 'repeat ...' 
Not consistent. Background not highlighted when viewed in html.